### DISCUSSION TITLE
How to build the non-beta version from source code

### DISCUSSION THREAD

**Discussion by gaurakshay** [Mimetis/Dotmim.Sync#525] [Q&A] [answered]
Hi @Mimetis !

This is following our discussion in #477 . I am building nuget packages for our private feed that changes the dependencies of the `SQLite` library to work with encrypted db and is working fine so far.

My question is how do I build the latest non-beta version of the app? For example, I was building the package from the current `master` branch and it builds a v7.4 which is still in beta. If I wanted the non-beta, I would have to go back to v7.3. 

I do see that there is historical commit that refers to v7.3. I am specifically asking in the context of CI CD pipeline which builds automatically whenever master is updated resulting in a new nuget package release. How do you handle your CI CD setup?

Any suggestions are appreciated :)

**Comment by Mimetis** [comment, marked as answer]
Each time I push a new release, I'm tagging the master branch with the corresponding version
So, if you want to go back to `v0.7.3`, just go back the last commit tagged `v0.7.3`

my **CI** is launched every time I have a new commit on the master branch, but the **CD** packages are built on demand

**Reply by gaurakshay** [reply]
Thank you!!!
